pulsar-function-go/go.mod

```diff
@@ -3,15 +3,16 @@ module github.com/apache/pulsar/pulsar-function-go
 go 1.13
 
 require (
-	github.com/apache/pulsar-client-go v0.7.0
-	github.com/golang/protobuf v1.4.3
-	github.com/prometheus/client_golang v1.7.1
+	github.com/apache/pulsar-client-go v0.8.1
+	github.com/dgrijalva/jwt-go v3.2.0+incompatible // indirect
```

Comment:
There's an existing advisory about this library: https://github.com/advisories/GHSA-w73w-5m7g-f7qc . Can we migrate to https://github.com/golang-jwt/jwt ?

Reply:
https://github.com/apache/pulsar-client-go/blob/3e1b39dd6f88aa0a12b949b1d1e064bde6355e22/oauth2/go.mod#L7 seems to be used in pulsar-client-go/oauth2

Reply:
The Go SDK does not directly reference this lib. This is indirectly referenced by other packages, and we cannot modify it.

Reply:
Hello @lhotari i send a pull request to Go SDK, refer to: https://github.com/apache/pulsar-client-go/pull/759, please check.

Reply:
@lhotari PTAL again, thanks